### PR TITLE
add dump, remove scaling from script

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:

* rescaling of the worker nodes and enabling nested virtualization has been [moved to the custom-test-image template](https://github.com/openshift/release/pull/7457).
* dump tool compilation has been added
* selection of high prio tests to mitigate the 4h timeout restriction

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dhiller** [comment]
/hold

need to wait until other PR has been merged

openshift/release#7290

File: hack/ci/test.sh [hunk 1]
```diff
@@ -1,60 +1,32 @@
...
-while ! "$nodes_ready"; do sleep 5 && if ! oc get nodes | grep NotReady; then nodes_ready=true; fi; done
-oc project default
-oc apply -f /go/src/kubevirt.io/kubevirt/kvm-ds.yml
-workers=$(oc get nodes | grep worker | awk '{ print $1 }')
-workers_each=($workers)
-for i in {0..2}; do
-    if ! oc debug node/"${workers_each[i]}" -- ls /dev/kvm; then oc debug node/"${workers_each[i]}" -- ls /dev/kvm; fi
-done
+
+export DOCKER_PREFIX='dhiller'
```
Comment (stu-gott) [inline hack/ci/test.sh]: For testing? Is the intent for this docker tag to be used long term?

Comment (dhiller) [reply hack/ci/test.sh]: No, this is only for testing.

Comment (dhiller) [reply hack/ci/test.sh]: I'm currently working to get the other PR in and then work on the details, i.e. building everything from scratch.

**Comment by dhiller** [comment]
/hold cancel

After I thought about it, it would be better if this one got merged before, so I could check whether the new changes in the other PRs work.

**Comment by dhiller** [comment]
Again, this one is required for kubevirt/kubevirt#3157 to work.

File: hack/ci/test.sh [hunk 2]
```diff
@@ -1,60 +1,32 @@
...
+export DOCKER_TAG="latest"
+export KUBEVIRT_PROVIDER=external
+
+oc config view
+
 echo "calling cluster-up to prepare config and check whether cluster is reachable"
+# TODO: remove patching of external provider, after kubevirtci#199 has been merged and kubevirt updates
+(
+    cd cluster-up/cluster/external &&
+        curl -L -O https://raw.githubusercontent.com/dhiller/kubevirtci/fix-external-provider/cluster-up/cluster/external/provider.sh
```
Comment (rmohr) [inline hack/ci/test.sh]: Would be great if kubevirtci#199 could be merged.

Comment (dhiller) [reply hack/ci/test.sh]: Has been done, removed.

**Pull request closed without merging**